CONTRIBUTING.md

```diff
@@ -150,6 +151,85 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Profiling
+Profiling can be used to monitor the impact of Secretless on CPU and Memory consumption. Currently, Secretless supports two types- CPU and Memory.
+
+**Prerequisites:**
+- [Graphviz](https://graphviz.gitlab.io/download/) to visualize profiling results
+- [Postgresql](https://www.postgresql.org/download/) to install Postgres
+
+In order to configure individual profiles properly and receive desired results, perform the following:
+1. [Build](https://conjurhq.slack.com/archives/G7XCMCEAV/p1544522629029800) Secretless locally
+2. Run a Postgres backend named ```sample-pg```:
+```
+pushd test/pg_handler
+  docker build -t sample-pg -f Dockerfile.pg .
+  docker run -d -p 5432:5432 sample-pg
+popd
+```
+
+3. Check if Postgres is running and query the database:
+```
+psql -h localhost -p 5432 -U test dbname=postgres
```

Comment:
it's probably easier to just append `-c "select count(*) from test.test;"` to the end of this command; then they can see if it works in one step instead of 3.

Reply:
up to you whether you want to make this change.